inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/embedding_bag_offsets_sum.cpp

```diff
@@ -11,18 +12,26 @@ using namespace LayerTestsDefinitions;
 
 namespace {
 
-const std::vector<InferenceEngine::Precision> netPrecisions = {
-        InferenceEngine::Precision::FP32,
-        InferenceEngine::Precision::I32,
-        InferenceEngine::Precision::U8
+const std::vector<ov::test::ElementType> netPrecisions = {
+        ov::test::ElementType::f32,
+        ov::test::ElementType::i32,
+        ov::test::ElementType::u8
 };
 
-const std::vector<InferenceEngine::Precision> indPrecisions = {
-        InferenceEngine::Precision::I64,
-        InferenceEngine::Precision::I32
+const std::vector<ov::test::ElementType> indPrecisions = {
+        ov::test::ElementType::i64,
+        ov::test::ElementType::i32
+};
+
+const std::vector<ov::test::InputShape> input_shapes = {
+        {{ov::Dimension::dynamic(), ov::Dimension::dynamic()}, {{5, 6}, {10, 35}}},
```

Comment:
please add comments where is dynamic and where is target shapes
https://github.com/openvinotoolkit/openvino/blob/c8b44b275cb11eed74e6ded64ab5f2afbf1cd21d/inference-engine/tests/functional/plugin/cpu/single_layer_tests/eltwise.cpp#L401-L413

Reply:
were added, thanks